### ISSUE TITLE
Spot nodes being shown as on-demand in real-time resource UI

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#15]
From: @mdaniel the "Usage Type" in the /resources.html dialog shows "On-demand" for a spot instance.

**Comment by dwbrown2** [comment]
I was able to successfully reproduce. This uncovered several issues and raised another product question. 

*The issues*: Spot nodes were being "accurately" detected for pricing purposes (when Spot Instance Tag Name & Value are set appropriately). However, the usage type of these nodes (i.e. Spot) were not being reported correctly in the dialog view. A change for this issue has now been push to frontend master. This also highlights a weakness in the Getting Started documentation for Business customers. There's a todo to improve on-boarding documentation around configuring Spot Instance Tag Names. 

*The question*: real-time assets are meant to provide immediate insight into run-rate costs for currently provisioned resources. Now that we have integrated with the AWS spot feed, we have access to much more accurate cost data for Spot nodes, before it was just a user-provided estimate. However, this AWS feed data is often delayed 1hr+. I see at least two potential options: 1) keep current approach (real-time data) / do not use spot feed data or 2) integrate with spot feed and have more accurate data delayed 1hr+. I'm in favor of # 1 as I think it's consistent with the value this view aims to provide. The cost allocation page is integrated with the cost_model and provides this delayed data with higher accuracy.
